### DISCUSSION TITLE
Domotizzare Cappa

### DISCUSSION THREAD

**Discussion by Cusu95** [indomus/forum#23] [GENERALE - Discussioni generiche]
Ciao a tutti
Avrei avuto la malsana idea di domotizzare la cappa della cucina (ho visto un utente del forum farlo).

L’idea sarebbe di sfruttare un sonoff iFan03 per controllare le luci e le tre velocità.

Purtroppo ho provato ma ho bruciato la scheda elettronica 😅 (per fortuna la garanzia mi copre anche danni accidentali).

Avete qualche consiglio/guida anche in inglese?

Grazie mille

**Comment by nromagna** [comment]
Mi associo alla richiesta.

**Comment by sky7176** [comment]
Ciao,
ma come hai collegato la tua cappa al sonoff iFan03?

Da quello che capisco,il sonoff iFan03 in uscita ha la tensione a 230 Vacc già regolata a seconda della velocità che imposti.

Credo che senza manomettere in mdo pesante la cappa il sonoff iFan03 non sia l'oggetto più adatto.
Forse potresti guardare il sonoff 4chpro, che ha 4 relè in uscita a contatti puliti, con lo schema alla mano della tua cappa, visto che ne hai già fritta una, andare a vedere di mettere le uscite deli relè in parallelo ai comandi, in pratica simuli quanto presente sulla cappa stessa.

Ci sarebbe da tenere conto anche le sequenza di attivazione/disattivazione dei vari relè, perchè non credo che alla cappa "piaccia" ricevere contemporaneamente, per esempio, sia il comando ventola minima che il comando ventola media o massima.

La mia cappa ha per la regolazione con un commutatore lineare con cui posso passare alle tre diverse velocità, ma torno a ripetere, prima ci vuole lo schema di collegamento della cappa e capire come lavora l'elettronica della cappa.

A titolo puramente di esmpio, non dare il mio indirizzo all'assistenza se ne friggi un'altra HI, io farei come da schema allegato.

Ma ricorda prima studia come lavora la cappa.

![Esempio](https://user-images.githubusercontent.com/13632506/113551462-f9c61800-95f4-11eb-8f16-84f8f61b63d1.png)


Giuseppe

**Reply by Cusu95** [reply]
Ciao!
Ho provato collegando l’uscita “fan” alle varie uscite della cappa, ma devo aver toccato qualche filo che non dovevo.
Sarebbe una buona soluzione anche il sonoff 4ch. Avevo pensato all’Iran associando il fatto che il motore della cappa è come se fosse una ventola, ma la tua soluzione credo sia più appropriata. 
Purtroppo non riesco a trovare da nessuna parte lo schema elettrico.
Proverò con un sonoff 4ch e chiamando un elettricista prima di fare altri danni😅
